sql/core/pom.xml

```diff
@@ -87,6 +87,16 @@
     </dependency>
 
     <dependency>
+      <groupId>org.apache.orc</groupId>
+      <artifactId>orc-core</artifactId>
+      <classifier>${orc.classifier}</classifier>
```

Comment:
sorry a dumb question, what does `classifier` mean? I don't see it in the rest of this pom file